biblib/views/user_view.py

```diff
@@ -58,9 +58,24 @@ def retrieve_user_email(owner_absolute_uid):
                     service
                 )
         return response
+    
+    @classmethod
+    def get_user_libraries(cls, session, service_uid, sort_col, sort_order, type):
+
+        query = session.query(Permissions, Library)\
+        .join(Permissions.library)\
+        .filter(Permissions.user_id == service_uid) 
+
+        if type == 'owner':
+            query = query.filter(Permissions.permissions['owner'].astext.cast(Boolean).is_(True))
+        elif type != 'all': 
```

Comment:
Is there a particular reason you opted for `!='all'` instead of `=='collaborator'`?